lib/components/viewers/viewers.css

```diff
@@ -102,48 +102,71 @@
 
 /* stop viewer styles */
 
-.otp .stop-viewer .route-row {
-}
 .otp .stop-viewer .route-row-container,
 .related-panel-container {
-  border-radius: 10px;
   box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.1);
   list-style-type: none;
   padding-left: 0;
 }
+.otp .stop-viewer .list-container .route-row-container li .header.stop-view {
+  border-radius: 0;
+} 
+
+.otp .stop-viewer .list-container .day-label {
+  font-weight: 500;
+}
+
+.otp .stop-viewer .list-container:first-of-type .route-row-container li:first-of-type .header.stop-view {
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+}
+.otp .stop-viewer .list-container:last-of-type .route-row-container li:last-of-type .header.stop-view  {
+  border-bottom-left-radius: 10px;
+  border-bottom-right-radius: 10px;
+}
+
 .otp .stop-viewer .route-row:first-of-type {
   margin-top: 10px;
 }
 
 .otp .stop-viewer .route-row .header {
   align-items: center;
-  display: grid;
-  grid-template-columns: 2fr 1fr;
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  overflow: hidden;
   width: 100%;
 }
 .otp .stop-viewer .route-row .header .route-name {
   align-items: center;
   display: grid;
   font-size: 42px;
   gap: 5px;
-  grid-template-columns: 4ch 2fr;
+  grid-template-columns: fit-content(8ch) fit-content(8ch);
```

Comment:
This causes the grid to sometimes flow strangely, with destinations being unaligned. I think we need a better solution here